### PR TITLE
chore: Release 0.2.0

### PR DESCRIPTION
**Pull request by JirkaChadima**
- Merge into master
- Manually tag merge commit with `v0.2.0` tag
- Push tag onto github. That should trigger Travis which should trigger now.sh deployment and cloudflare cache bust. After the job is completed, you should immediately see a new version in `meta` tags on demo.windingtree.com

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,4 +13,8 @@ jobs:
...
+  - secure: NVwrcw1HqG0kK3/YpLDh0fLb47oOo3nrdHzLc72T3PSugjucktMb6jDB9OJuCwGCvievEsbxSdNZThmR1FRVO4lkAoM6HfHbKKTK72hQH0oFc9exBzXaMUgF30nxHVsb89gP5BNnC3DEcV2eFBcBm7PoS90mOI99umSI+IbU6Sq3WpkjV05wP5vBxuuImpyI8m6Nb78LlGotrCaf+7auHVudfHIAH/qJRoYQKAGWnbD8VnP1/hJWPDXJ37+W1b5iv9kcKFP3cbDysEbvivb5W8Mykm0zhJo3QGv/gk5pwmYdFV6PmpwZhPBblHJqaY7U+CJxO1GMaxQLoYGZuDLqsvodB7ToQYh6nQmT91eSH51M7WoIbrzv0FlwLXTKSisMTfqUYV79ysMxb+u9Tdry2hDG4xYPgmC/L73z6YAK9eQK9z8qZJDhCsgr/H56gqoFR8GeeOqGL6wck+UCcg6isloj9zRcRiDY686YKDW+SyAuYvTj8NgjFojSE8e7X+qFOtgKqjaym1Bo/4og0I0YpbTuOYAw/5gDVNSOKlOtBgq7khqH2IsBluYcYaProsdNmmfEviBv8lZfpOU3ccxk8xPmCbQ0GolUC+WTU5dQeDhbzUjVizUEjzaisH8Neu4YqmgyWKxMdYZ3t5cRK2FX3slG7RKP69B6TR4UIH4qCSc=
```
Comment (AugustoL) [inline .travis.yml]: why so many credentials? we only need one right?

Comment (kvbik) [reply .travis.yml]: it is not a credential. it is couple of encrypted ENV variables. many of them are for the cloudflare  wipe script.

Comment (kvbik) [reply .travis.yml]: it would be great if we can comment  those here....

Comment (JirkaChadima) [reply .travis.yml]: I have added some claryfying comments. However, iIf it's not commented, the order of secure variables seems to be maintained in Travis build log `Setting environment variables from .travis.yml` section

**Comment by JirkaChadima** [comment]
@kvbik Yes, it's already deployed (done that during testing of the cloudflare script), only with a bad commit hash and version tag in `meta` section of the page. That should be fixed after merging this.

**Pull request merged**